assets/embauche.html

```diff
@@ -50,7 +50,16 @@
 					<option>mois</option>
 					<option disabled>jour</option>
 					<option disabled>heure</option>
-				</select>.
+				</select>
+
+				sur la commune de
+				<input name="code_postal_entreprise" id="code_postal_entreprise" class="medium-width centered"
+				type="text" placeholder="code postal" pattern="\d{5}" title="Entrez le code postal de l'établissement où le salarié sera employé">
```

Comment:
A `placeholder` should present the user with a valid input example, not a field name or an instruction, which is meant for the `title`.